front/src/modules/ui/tooltip/__stories__/Tooltip.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { userEvent, within } from '@storybook/testing-library';
+
+import { CatalogDecorator } from '~/testing/decorators/CatalogDecorator';
+import { ComponentDecorator } from '~/testing/decorators/ComponentDecorator';
+
+import { AppTooltip, TooltipValues } from '../AppTooltip';
+
+const meta: Meta<typeof AppTooltip> = {
+  title: 'UI/Tooltip/Tooltip',
+  component: AppTooltip,
+};
+
+export default meta;
+type Story = StoryObj<typeof AppTooltip>;
+
+export const Default: Story = {
+  args: {
+    place: 'bottom',
+    content: 'Tooltip Test',
+    anchorSelect: '#hover-text',
+  },
+  decorators: [ComponentDecorator],
+  render: (args) => (
+    <>
+      <p id="hover-text" data-testid="tooltip">
+        Hover me!
+      </p>
+      <AppTooltip {...args} />
+    </>
+  ),
+  play: async ({ canvasElement }) => {
```

Comment:
please user parameters: { hover: [.hover] }
See Chip story for example.

This is a cleaner way and does not require to add a render function and a play test.

We don't need to how the hover in the Default Story, it's good to have it in the Catalog one